client/sites/hooks/use-initialize-breadcrumbs.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'calypso/state';
+import { resetBreadcrumbs } from 'calypso/state/breadcrumb/actions';
+
+export function useInitializeBreadcrumbs( siteId: number | undefined ) {
+	const dispatch = useDispatch();
+	useEffect( () => {
+		if ( ! siteId ) {
```

Comment:
There is still an issue with this. For example,
1. Sites
2. Pick a site
3. Settings
4. Reset site
5. Click browser back button to go back to the site overview
6. Plugins
7. Pick a plugin
8. Click browser back button twice to go back to the site overview again
9. Boom!

Reply:
Oops, thanks for catching, let me think of a more robust solution... 🙏